management/src/test/java/org/ehcache/management/cluster/ClusteringManagementServiceTest.java

```diff
@@ -241,7 +242,7 @@ public void test_stats_collection() throws Exception {
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
     assertThat(stats[0].getStatistic(Counter.class, "GetCounter").getValue(), equalTo(4L));
     assertThat(stats[0].getContext().get("cacheName"), equalTo("cache-1"));
-    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, equalTo(1));
+    assertThat(stats[0].getStatistic(CounterHistory.class, "AllCacheGetCount").getValue().length, greaterThanOrEqualTo(1));
```

Comment:
@mathieucarbou Given the explanation above, and the original question, if the fix had contained a modification on line 246 (new version) so that the assertion verified that the _last value_ was `4`, it would have made sense.